org-cyf-itd/content/blocks/step-1/instructions/index.md

```diff
@@ -57,7 +57,7 @@ Software for Lawyers Ltd makes case management software. Their software helps la
 5. Create a Google doc with the text of the cover letter you generated. If this platform is new to you, there is this [guide on what Google docs is and how to use it](https://support.google.com/docs/answer/7068618?hl=en-GB&co=GENIE.Platform%3DDesktop) available.
 6. Please put your name in the title of the Google doc and **make it public**. You can add just your given name or your family name if you prefer.
 7. Add the link to the Google doc to this platform
-8. Add a comment with at least three comments (positive or negative) on the AI’s work
+8. Add a comment on the Google Doc with feedback (positive or negative) on the AI’s work. This comment should be related to the content of the document. It should not be a general comment about AI or your feelings about using AI. It should evidence that you have thought critically about the AI's work.
```

Comment:
```suggestion
8. Add a comment on the Google Doc with feedback (positive or negative) on the text the AI has generated. It should not be a general comment about AI or your feelings about using AI; it should be your own evaluation of the text you are submitting. 
```

Reply:
In last ITD, some partcipants literally left their comments as comments (instead of as text) in their Google doc or on Dashboard. Those comments are not visible to public by default, so volunteers cannot see them and repeatedly rejected their submissions and kept asking them to leave comments even though from the participants' view they had already left the comments. Can we state clearly where to left comments? I think in the Google doc as text after the cover letter is good.